build/Targets/VSL.Versions.targets

```diff
@@ -69,6 +69,25 @@
     </Otherwise>
   </Choose>
 
+  <Choose>
+      <When Condition="'$(BUILD_SOURCEVERSION)' != ''">
+          <PropertyGroup>
+              <GitHeadSha>$(BUILD_SOURCEVERSION)</GitHeadSha>
+          </PropertyGroup>
+      </When>
```

Comment:
Need to add a similar `When` as above when `GIT_COMMIT` is a non-empty value.  That is the commit SHA that Jenkins passes down to us.  

Hmm.  Overall I think we may want to change this to also respect when `GitHeadSha` is expliictly set a value at the msbuild command line.